liquidprompt

```diff
@@ -1594,6 +1595,23 @@ _lp_as_text()
     echo -nE "$1" | sed -$_LP_SED_EXTENDED "s,$_LP_CLEAN_ESC,,g"
 }
 
+# Get length of a given string, after evaluating escape codes (\u for user, etc)
+_lp_length()
+{
+    typeset len evaluated_ps1
+    typeset printable_ps1=$(_lp_as_text "$1")
```

Comment:
This (typeset and assignment on the same line) is unsafe in zsh < 5.1:
```zsh
% func() { 
  typeset var=$(echo foo bar)
  echo "$var"
}
% func
foo
```
Either wrap in quotes or break into two lines:
```suggestion
    typeset printable_ps1="$(_lp_as_text "$1")"
```
Check for other issues like this, I see a few other instances.